apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,367 @@
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                ReadModifyWriteRules.Increment("Score", "Level 1", 55));
+
+            Cell modifiedCell = response.Row.Families[0].Columns[0].Cells[0];
+
+            // Implicitly convert the ByteString Value into a BigtableByteString,
+            // which supports additional conversions specific to Bigtable.
+            BigtableByteString value = modifiedCell.Value;
+
+            // Convert the BigtableByteString to a long to obtain the updated value.
+            long newScore = (long)value;
+
+            // End snippet
+
+            Assert.Equal(67, newScore);
+        }
+
+        [Fact]
+        public async Task ReadRowAsync()
+        {
+            var cts = new CancellationTokenSource();
+            var cancellationToken = cts.Token;
+            var client = _fixture.TableClient;
+            var projectId = _fixture.TableName.ProjectId;
+            var instanceId = _fixture.TableName.InstanceId;
+            var tableId = _fixture.TableName.TableId;
+
+            await client.MutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                Mutations.DeleteFromRow(),
+                Mutations.SetCell("Address", "Country", "USA"),
+                Mutations.SetCell("Address", "Line 1", "1234 Main Street"),
+                Mutations.SetCell("Address", "Line 2", "Apt. 2"),
+                Mutations.SetCell("Score", "Level 1", 12));
+
+            // Snippet: ReadRowAsync(TableName,BigtableByteString,RowFilter,CallSettings)
+            // Only read cells from the "Line 1" and "Line 2" columns in the
+            // Address column family and only take the most recent cell from
+            // each column.
+            Row row = await client.ReadRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                RowFilters.Chain(
+                    RowFilters.FamilyNameRegex("Address"),
+                    RowFilters.ColumnQualifierRegex(@"Line \d"),
+                    RowFilters.CellsPerColumnLimit(1)),
+                CallSettings.FromCancellationToken(cancellationToken));
+
+            Family addressFamily = row.Families.FirstOrDefault();
+            if (addressFamily != null)
+            {
+                foreach (Column column in addressFamily.Columns)
+                {
+                    Console.WriteLine($"    Column qualifier: {column.Qualifier.ToStringUtf8()}");
+                    foreach (Cell cell in column.Cells)
+                    {
+                        Console.WriteLine($"      Cell value: {cell.Value.ToStringUtf8()}");
+                        Console.WriteLine($"      Timestamp: {cell.Version.ToDateTime()}");
+                        Console.WriteLine($"      Labels: {string.Join(", ", cell.Labels)}");
+                    }
+                }
+            }
+
```

Comment:
Remove blank line at the end of the snippet? We may do that automatically, but I'm not sure.

Reply:
Done